seed/data_importer/views.py

```diff
@@ -539,25 +539,6 @@ def retrieve(self, request, pk=None):
         if not import_file.uploaded_filename:
             f['uploaded_filename'] = import_file.filename
         f['dataset'] = obj_to_dict(import_file.import_record)
-        # add the importfiles for the matching select
```

Comment:
Note that these are `importfiles` of a dataset (ImportRecord) of an ImportFile. For example, given an ImportFile `test.xlsx`, these are the ImportFiles belonging to the same ImportRecord that the ImportFile `test.xlsx` belongs to.

I haven’t been able to trace any FE or BE code that uses those specific `importfiles`. Upon removing them from the request, the bug is fixed, all tests still pass, and monkey testing importing/matching files all seem to be OK.

Note that this `importfiles` logic seems to have originated in the initial commit of the app https://github.com/SEED-platform/seed/blame/cd6edee5778d7d117e5469e4cc8a875bf7093259/seed/views/main.py#L1566

I suspect it might be remnants of an old architecture.